src/plugins/proxy/dev_api/openvino/proxy/properties.hpp

```diff
@@ -0,0 +1,59 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <string>
+
+#include "openvino/runtime/properties.hpp"
+
+namespace ov {
+namespace proxy {
+
+// Proxy plugin configuration properties
+namespace configuration {
```

Comment:
We use properties under this namespace only for the proxy configuration. 
It is why these properties are here, in order to avoid a mix with proxy plugin runtime properties.